build.gradle

```diff
@@ -259,6 +265,6 @@ task dist(dependsOn: assemble) {
 
 task wrapper(type: Wrapper) {
 	description = 'Generates gradlew[.bat] scripts'
-	gradleVersion = '2.5'
+	gradleVersion = '2.11'
```

Comment:
Why not `2.12` already?
Ah! Yes, that is from SK.
As you see I followed there with the Rob Winch's notice that we don't need the `wrapper` task just to keep the Gradle version. There is just enough to change the version in the `distributionUrl` variable in the `gradle-wrapper.properties`.